OrbitGl/SliderTest.cpp

```diff
@@ -14,48 +14,271 @@ class MockCanvas : public GlCanvas {
   MOCK_METHOD(int, GetHeight, (), (const, override));
 };
 
-TEST(Slider, Dragging) {
-  using ::testing::Return;
-  const float kEpsilon = 0.01f;
+template <int dim>
+void pick(GlSlider& slider, int start, int other_dim = 0);
 
-  MockCanvas canvas;
+template <>
+void pick<0>(GlSlider& slider, int start, int other_dim) {
+  slider.OnPick(start, other_dim);
+}
+
+template <>
+void pick<1>(GlSlider& slider, int start, int other_dim) {
+  slider.OnPick(other_dim, start);
+}
```

Comment:
`if constexpr(dim == 0) {` might make this cleaner. WDYT?

Reply:
Good point, thanks for the suggestion.